package-lock.json

```diff
@@ -1,7517 +1,8 @@
 {
 	"name": "asciidoctor-vscode",
-	"version": "0.0.0",
-	"lockfileVersion": 2,
+	"version": "2.9.0",
+	"lockfileVersion": 1,
```

Comment:
Could you please revert this change? Alternatively, you can use npm >= 7 to regenerate the `package-lock.json` file (using version 2 format).